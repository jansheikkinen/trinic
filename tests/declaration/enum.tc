// Enum

enum TestEnum
  ENUM_ONE,
  ENUM_TWO = 2,
  ENUM_THREE
end
