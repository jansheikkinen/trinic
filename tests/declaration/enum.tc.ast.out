
### AST GENERATION ###
Declaration ENUM
  Arg ASSIGN
    Assign_Arg ENUM_ONE
    Assign_Arg ENUM_TWO
      Expr LITERAL
        INT 2
    Assign_Arg ENUM_THREE
